File: engine/model/src/main/scala/net/server/channel/packet/mts/SearchPacket.scala
package net.server.channel.packet.mts

class SearchPacket(private var _available: Boolean, private var _operation: Byte, private var _tab: Int, private var _type: Int, private var _ci: Int, private var _search: String) extends BaseMTSPacket(_available = _available, _operation = _operation) {
  def tab: Int = _tab

  def theType: Int = _type

  def ci: Int = _ci

  def search: String = _search
}

File: engine/engine-model/src/main/java/net/server/channel/packet/mts/SearchPacket.java
package net.server.channel.packet.mts;

public class SearchPacket extends BaseMTSPacket {
   private final Integer tab;

   private final Integer theType;

   private final Integer ci;

   private final String search;

   public SearchPacket(Boolean available, Byte operation, Integer tab, Integer theType, Integer ci, String search) {
      super(available, operation);
      this.tab = tab;
      this.theType = theType;
      this.ci = ci;
      this.search = search;
   }

   public Integer tab() {
      return tab;
   }

   public Integer theType() {
      return theType;
   }

   public Integer ci() {
      return ci;
   }

   public String search() {
      return search;
   }
}
